tests/test_sanitize.py

```diff
@@ -114,3 +118,70 @@ def test_error_annotation_all():
                               {'name': 'Multifurcation', 'label': 'Circle8', 'color': 'Yellow',
                                'data': [(0, np.array([[0., 5., 0., 1.]], dtype=np.float32))]}],
         str(morph_paths[1]): []})
+
+
+@contextmanager
+def _tmp_file(content, extension):
+    with tempfile.NamedTemporaryFile(suffix='.' + extension, mode='w') as tmp_file:
+        tmp_file.write(content)
+        tmp_file.flush()
+        tmp_file.seek(0)
+
+        yield tmp_file
+
+        tmp_file.close()
```

Comment:
ups, I forgot to remove this... It was just to test what I should put in the file.